composer.json

```diff
@@ -19,7 +20,8 @@
         "symfony/framework-bundle": ">=3.4",
         "symfony/http-foundation": ">=3.4",
         "symfony/routing": ">=3.4",
-        "symfony/yaml": ">=3.4"
+        "symfony/yaml": ">=3.4",
+        "symfony/templating": ">=3.4"
```

Comment:
Actually templating is [deprecated since 4.3](https://symfony.com/blog/new-in-symfony-4-3-deprecated-the-templating-component-integration).